tests/test_trackingstore.py

```diff
@@ -1017,3 +1008,41 @@ def test_restore_client_error(mocker):
         store.restore_experiment(EXPERIMENT_ID)
 
     mock_client.restore.assert_called_once_with(PROJECT_ID, EXPERIMENT_ID)
+
+
+def test_invalid_experiment_id(mocker):
+    mock_client = mocker.Mock()
+
+    mocker.patch("faculty.client", return_value=mock_client)
+
+    store = FacultyRestStore(STORE_URI)
+
+    with pytest.raises(ValueError):
+        store.create_run(
+            "invalid-experiment-id",
+            "unused-mlflow-user-id",
+            RUN_NAME,
+            "unused-source-type",
+            "unused-source-name",
+            "unused-entry-point-name",
+            RUN_STARTED_AT,
+            "unused-source-version",
+            [MLFLOW_TAG],
+            PARENT_RUN_UUID_HEX_STR,
+        )
+
+    with pytest.raises(ValueError):
```

Comment:
We'd usually have separate tests for each of the affected methods, i.e.:

* test_get_experiment_invalid_id
* test_delete_experiment_invalid_id
* test_restore_experiment_invlaid_id
* etc.

Reply:
Yes, good catch, I had not realised how many methods were affected. I have added tests for all cases (I think)